image_data_guide/04a_builtin_training.ipynb

```diff
@@ -0,0 +1,487 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Training Built-in Algorithms with SageMaker (Part 4/4)\n",
+    "Download | Structure | Preprocess | **Train**\n",
+    "```\n",
+    "\n",
+    "```\n",
+    "Note: this notebook should be used with the conda_amazonei_mxnet_p36 kernel"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<pre>\n",
+    "</pre>"
+   ]
+  },
+  {
```

Comment:
For consistency, add a TOC like you did in the other notebooks?

Reply:
TOC with hyperlinks added.